livekit-agents/livekit/agents/worker.py

```diff
@@ -583,7 +589,11 @@ async def _on_accept(args: JobAcceptArguments) -> None:
 
         logger.info(
             "received job request",
-            extra={"job_request": msg.job, "resuming": msg.resuming},
+            extra={
+                "job_request": msg.job,
+                "resuming": msg.resuming,
+                "agent_name": self._opts.agent_name,
```

Comment:
I think if the worker can only have one agent_name. It isn't useful to relog it every time?

Reply:
IMO actually we should log it even more...for example, imagine you wanted to do a datadoq query against specific agent_names. Maybe you want to see if VAD takes too long on certain agent_names

Reply:
I would use the WorkerID no?
all the logs coming from the child processes will not print agent_names anyway.
Think we should add worker_id everywhere tho

Reply:
I think it would be useful to have agent_name as well. At least I can imagine it being valuable for agent_name to indexed in logging platforms (Datadogs, ELK, w/e). For example, being able to see that a bunch of error logs only happen on a specific agent_name would be useful to know

Reply:
agreed. each log line should have sufficient context for a search query. context is super helpful when users are searching.